src/components/SozialMarie/index.jsx

```diff
@@ -0,0 +1,208 @@
+import useTimer from 'hooks/useTimer';
+import { Alert, Box, Divider, Snackbar, Stack } from '@mui/material';
+import { useLocalStorage } from 'hooks';
+import { useEffect, useState, useCallback, useRef } from 'react';
+import i18n, { t } from 'i18next';
+import VotingButton from './VotingButton';
+import AlertCountDown from './AlertCountDown';
+import SozialMarieLink from './SozialMarieLink';
+import AlertFooterContent from './AlertFooterContent';
+import AlertContentHeader from './AlertHeaderContent';
+import { startDate, endDate, doNotShowBefore } from './date-range';
+import { LOCAL_STORAGE_KEY, LOCAL_STORAGE_VALUES, getInitialIsShow } from './localStorage';
+
+const DELAY_TO_HIDE_ALERT = 5000;
+const DELAY_TO_HIDE_TRIGGER = 6000;
+
+if (DELAY_TO_HIDE_ALERT > DELAY_TO_HIDE_TRIGGER) {
+  throw new Error('DELAY_TO_HIDE_ALERT should be less than DELAY_TO_HIDE_TRIGGER');
+}
+
+const SOZIAL_MARIE_LINK = `https://www.sozialmarie.org/${i18n.language === 'it' ? 'en' : i18n.language}/projects/9280/`;
```

Comment:
Link is always https://www.sozialmarie.org/sl/projects/9280/ for some reason.

Reply:
this https://github.com/sledilnik/zdravniki/pull/474/commits/80a72fc8383e1c6f0a68878e49d750c26888f4b3  should fix link issue